src/commands/import-resolver.ts

```diff
@@ -77,12 +77,16 @@ export function createImportResolver({
       return spec;
     }
     if (dependencyImportMap && dependencyImportMap.imports[spec]) {
+      // if baseURL is remote, handle that outside of path.posix.join()
+      const PROTOCOL_REGEX = /^https?:\/\//;
```

Comment:
nit: Regex creation can be expensive, safe to move up to the top level

Reply:
Should this also handle protocol-less urls? `//`?

Reply:
> Should this also handle protocol-less urls? `//`?

Ooh, good catch! 👏